modules/cluster-logging-collector-log-forward-project.adoc

```diff
@@ -66,20 +67,23 @@ spec:
 <5> Specify the URL and port of the external log aggregator as a valid absolute URL. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP address.
 <6> If using a `tls` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and have *tls.crt*, *tls.key*, and *ca-bundle.crt* keys that each point to the certificates they represent.
 <7> Configuration for an input to filter application logs from the specified projects.
-<8> Configuration for a pipeline to use the input to send project application logs to an external Fluentd instance.
-<9> The `my-app-logs` input.
-<10> The name of the output to use.
-<11> Optional: String. One or more labels to add to the logs.
-<12> Configuration for a pipeline to send logs to other log aggregators.
-** Optional: Specify a name for the pipeline.
-** Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
-** Specify the name of the output to use when forwarding logs with this pipeline.
-** Optional: Specify the `default` output to forward logs to the internal Elasticsearch instance.
-** Optional: String. One or more labels to add to the logs.
+<8> If no namespace is specified, logs are collected from all namespaces.
+<9> Configuration for a pipeline to use the input to send project application logs to an external Fluentd instance.
```

Comment:
FYI: not sure if this is clearer but  - a pipeline directs logs from a named input to a named output. So the pipeline forwards logs to the output named "fluentd-server-insecure", it's the output definition that specifies this is going to fluentd. Maybe too much nitpicking.

Reply:
Made some updates to clarify.